packages/feedback-service/src/resolver.ts

```diff
@@ -1,70 +1,32 @@
+import * as _ from "lodash";
 import { Feedback } from './schema';
-import { getTransporter } from './helpers';
-  
-const transporter = getTransporter;
 
 export const FeedbackResolver = {
   Query: {
-    //queries
     listFeedback(root: any, args: any, ctx: any) {
       return Feedback.find()
         .sort({ "timestamp": -1 })
         .then(response => response)
         .catch(err => err);
     },
     getFeedback(root: any, args: any, ctx: any) {
-      return Feedback.findById(args._id)
+      return Feedback.findById(args.id)
         .then(response => response)
         .catch(err => err);
+    },
+    getFeedbackBy(root: any,  args : any, ctx: any){
+      const searchBy = _.pick(args, ["input"]);
```

Comment:
The `_.pick` is to pick out selected properties from an object. What you're doing here has no effect.
You need to pick out the properties from the `args.input` object. And use those properties in the find.

Reply:
Actually _.pick is not required here i tested return Feedback.find(args.input).exec(); like this mongoDB take care of it and i also printed filed there which ever fileds i am passing from browser only those field are there.
so i am removing _.pick from here.